wallet/txsizes/size.go

```diff
@@ -156,11 +195,11 @@ func EstimateSerializeSize(inputCount int, txOuts []*wire.TxOut, addChangeOutput
 }
 
 // EstimateVirtualSize returns a worst case virtual size estimate for a
-// signed transaction that spends the given number of P2PKH, P2WPKH and
+// signed transaction that spends the given number of P2PKH, P2TR, P2WPKH and
 // (nested) P2SH-P2WPKH outputs, and contains each transaction output
 // from txOuts. The estimate is incremented for an additional P2PKH
 // change output if addChangeOutput is true.
-func EstimateVirtualSize(numP2PKHIns, numP2WPKHIns, numNestedP2WPKHIns int,
+func EstimateVirtualSize(numP2PKHIns, numP2TRIns, numP2WPKHIns, numNestedP2WPKHIns int,
```

Comment:
TBH, we could prob make lnd's weight estimator into a module and use it here, then we wouldn't need to add a new arg here each time we have a diff output type (ofc at this pace it doesn't matter much). 

Just a note, don't think it's something we need to do rn.